doc/mediasdk-man.md

```diff
@@ -566,7 +566,7 @@ The application can use multiple SDK sessions independently or run a “joined
 To join two sessions together, the application can use the function [MFXJoinSession](#MFXJoinSession). Alternatively, the application can use the function [MFXCloneSession](#MFXCloneSession) to duplicate an existing session. Joined SDK sessions work
 together as a single session, sharing all session resources, threading control and prioritization operations (except hardware acceleration devices and external allocators). When joined, one of the sessions (the first join) serves as a parent session, scheduling execution resources, with all others child sessions relying on the parent session for resource management.
 
-With joined sessions, the application can set the priority of session operations through the [MFXSetPriority](#MFXSetPriority) function. A lower priority session receives less CPU cycles. Session priority does not affect hardware accelerated processing.
+With joined sessions, the application can set the priority of session operations through the [MFXSetPriority](#MFXSetPriority) function. A lower priority session receives less CPU cycles and make the hardware accelerater task priority lower during hardware accelerater scheduling.
```

Comment:
Please fix a typo: accelerater -> accelerator. Or better rephrase.

Reply:
hi Onabiull, yes I will modify it.

Reply:
Need to fix grammar as Oleg mentioned. 

"and explicitly sets the priority of hardware-accelerated tasks lower "

Reply:
Done.